src/app/components/filter/filter-panel.jsx

```diff
@@ -23,21 +23,34 @@ const useStyles = makeStyles({
     background: '#fafdff',
     display: 'grid',
     gridTemplateColumns: '1fr',
-    padding: '16px',
     rowGap: '16px',
+    maxHeight: '64vh',
     [theme.breakpoints.up('sm')]: {
       gridTemplateColumns: '1fr 1fr',
     },
   },
+  filters:{
+    height: '100%',
+    borderRight: '1px solid #599bde',
+    padding: '16px',
+    zIndex: 15,
```

Comment:
Why the zIndex? I thought this was used for things that were layered on top of each other, and I don't think any of these are